packages/massa-web3/src/experimental/errors/base.ts

```diff
@@ -0,0 +1,57 @@
+import { ErrorCodes } from './utils/codes'
+
+export type BaseErrorParameters = {
```

Comment:
Is this long name mandatory?

Reply:
This one is shorter. It's less descriptive but should be enough as it's only used here